app/src/ui/changes/no-changes.tsx

```diff
@@ -0,0 +1,30 @@
+import * as React from 'react'
+import { LinkButton } from '../lib/link-button'
+
+const BlankSlateImage = `file:///${__dirname}/static/empty-no-file-selected.svg`
+
+interface INoChangesProps {
+  /** Called when the user chooses to open the repository. */
+  readonly onOpenRepository: () => void
+}
+
+/** The component to display when there are no local changes. */
+export class NoChanges extends React.Component<INoChangesProps, void> {
+  public render() {
+    const opener = __DARWIN__ ? 'Finder' : 'Explorer'
+    return (
+      <div className='panel blankslate' id='no-changes'>
+        <img src={BlankSlateImage} className='blankslate-image' />
+        No local changes
+
+        <span>
```

Comment:
I think `p` is more appropriate here since this should appear on the line beneath _No Local Changes_. You may have to add a style to get it to display compactly though.